dataactbroker/README.md

```diff
@@ -408,6 +408,32 @@ Possible HTTP Status Codes:
 
 - 403: Permission denied, user does not have permission to view this submission
 
+#### GET "/v1/latest\_certification\_period/"
+This endpoint returns the latest certification period for the broker application.
+
+##### Sample Request
+`/v1/latest_certification_period/`
+
+##### Request Params
+N/A
+
+##### Response (JSON)
+```
+{
+    "quarter": 4,
+    "year": 2019
+}
+```
+
+##### Response Attributes
+- `quarter`: (integer) the quarter of the latest certification period, or none if no period is found
+- `year`: (integer) the fiscal year of the latest certification period, or none if no period is found
+
+##### Errors
+Possible HTTP Status Codes:
+
+- 403: Permission denied, user does not have permission to view this submission
```

Comment:
Why would `permission to view a submission` matter? Wouldn't this just be `401 - login required`?